QuizAppNew/View/QAview.swift

```diff
@@ -10,6 +10,12 @@ import SwiftUI
 //MARK: Question Answer view
 struct QAview: View {
     @State var isSubmitted = false
+    @ObservedObject var viewModel = QuestionViewModel()
+    //outgoing
```

Comment:
what is this **@ObservedObject var viewModel = QuestionViewModel()** why set it with **@ObservedObject**

Reply:
created intence for QuestionViewModel

Reply:
why set it? ObservedObject?

Reply:
@ObservedObject use koresi Karon class er protocol  @ObservableObject R @published variable use kora koresi jei class e JSON data store koresi okhane. @ObservedObject er vitor @ObservableObject class er data store koresi. R published diyesi cz kono property er vitor change hole  view update korbe ..@ObservedObjec view pass korbe ..